src/models/event.ts

```diff
@@ -1003,6 +1039,14 @@ export class MatrixEvent extends EventEmitter {
         return this.getType() === "m.room.redaction";
     }
 
+    public isVisibilityChange(): boolean {
```

Comment:
Visibility can mean all sorts of things, so I'd suggest we name this to avoid confusion with directory and room visibility.
```suggestion
    public isEventVisibilityChange(): boolean {
```